scripts/portfolioView.js

```diff
@@ -18,6 +18,10 @@ portfolioView.renderIndexPage = function() {
     $('#articles').append(article.toHtml());
   });
   portfolioView.handleMainNav();
+  $('#num-articles').text(Article.numArticles());
+  $('#avg-word-count').text(Article.avgWordCount());
+  // TODO: report avg post age
+  $('#avg-post-age').text(Article.avgPostAge());
```

Comment:
I like that I can easily tell what this does by reading the code.

An alternative way to this is to setup a handlebars template for the stats. Not necessary, but just another way to go about doing it.